check.py

```diff
@@ -14,30 +14,35 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import os, shutil, sys, subprocess, difflib, json, time, urllib2
+import os, shutil, sys, subprocess, difflib, json, time, urllib2, optparse
```

Comment:
We should probably use `argparse` instead of the obsolete `optparse`. I think it should be a pretty minor change from what we have here since it looks like we're just using the basic features.